docs/notebooks/regional_frequency_analysis.ipynb

```diff
@@ -0,0 +1,970 @@
+{
```

Comment:
<p><code>Explainatory</code> --&gt; <code>Explanatory</code> </p><p> </p><p>Aussi, je propose:</p><blockquote>Regional frequency analyses rely on explanatory variables to link the information at the various sites. For this example, we'll use catchment properties, but other variables sur as climatological averages or land use data could also be used. Refer to the GIS example for more details.</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/hydrologie/xhydro/pull/186/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>